ckanext/versioning/logic/action.py

```diff
@@ -455,3 +456,9 @@ def _get_dataset_name(id_or_name):
         raise toolkit.ObjectNotFound('Package {} not found'.format(id_or_name))
 
     return dataset.name
+
+
+def is_valid_revision_id(revision_ref):
```

Comment:
Yeah, this is good thinking but is not going to work with all backends (esp not with GitHub) because the ID of github commits is 40 hex digits (IIRC, SHA-1) and not 32 hex digits like local storage. In general and as a concept, revision IDs are opaque strings and depend on the backend. 

For this reason, I craeted this new PR in metastore-lib: https://github.com/datopian/metastore-lib/pull/27 - once it is merged, we can bump the version of metastore-lib required by ckanext-versioning to 0.2.0 and use this new API. It will always tell you if the string is a valid ID or not, without caring about backend specifics.

Reply:
I've released `metastore-lib` version `0.2.0` so you can now upgrade our requirements file to it and just do:

```python
if get_metastore_backend().is_valid_revision_id(revision_ref):
    # ...
```
Instead of this.